lib/components/switch/switch.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:f_kit_x/resource/theme_color.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_switch/flutter_switch.dart';
+
+class SwitchToggle extends StatefulWidget {
+  final bool isToggled;
+  final String label;
```

Comment:
Remove label add it as component implementation, so we wouldn't be dependent on this label if we ever want to use this component on other places